solr/core/src/test/org/apache/solr/util/SolrPluginUtilsTest.java

```diff
@@ -338,6 +339,18 @@ public void testMinShouldMatchCalculator() {
         
   }
 
+  @Test
+  public void testMinShouldMatchBadQueries() {
+    try {
+      calcMSM(1, "1<");
```

Comment:
could you please use `expecThrows()` here instead of try-catch

Reply:
Done.